test/e2e/scaffold/crd.go

```diff
@@ -22,14 +22,13 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/api7/ingress-controller/pkg/apisix"
```

Comment:
Style: put these lines at the end of import block.

Reply:
In the test mod , `github.com/api7/ingress-controller` is an externally imported package.
I used fmt tool for me to deal with group of imports.

Reply:
My bad.